CMakeLists.txt

```diff
@@ -7,6 +7,16 @@ include(CMakePackageConfigHelpers)
 option(BUILD_PYTHON "Build " ON)
 option(BUILD_STATIC "Build as a static library" OFF)
 
+if(WIN32)
+  set(BUILD_WITH_THREADS_DEFAULT OFF)
+else()
+  set(BUILD_WITH_THREADS_DEFAULT ON)
+endif()
+option(BUILD_WITH_THREADS "Whether to link against threads or not." ${BUILD_WITH_THREADS_DEFAULT})
+
+if(BUILD_WITH_THREADS)
+    find_package(Threads)
```

Comment:
I think it's more robust to do `find_package(Threads REQUIRED)` here. If it's not found, the line lower `target_link_libraries(Savitar PUBLIC pthread pugixml)` will fail.

Reply:
Makes sense 👍